src/transaction/boot_cl.c

```diff
@@ -5477,7 +5647,8 @@ catcls_vclass_install (void)
     {"CTV_STORED_PROC_NAME", boot_define_view_stored_procedure},
     {"CTV_STORED_PROC_ARGS_NAME", boot_define_view_stored_procedure_arguments},
     {"CTV_DB_COLLATION_NAME", boot_define_view_db_collation},
-    {"CTV_DB_CHARSET_NAME", boot_define_view_db_charset}
+    {"CTV_DB_CHARSET_NAME", boot_define_view_db_charset},
+    {CTV_DB_SERVER_NAME, boot_define_view_db_server}
```

Comment:
In the catcls_class_install() function, the macro variable value is used as the name value in the clist array. How about using the same macro variable for code unity?

Reply:
Unlike the catcls_class_install() function, the catcls_vclass_install() function does not use the name item of the clist structure.
However, semantically, if you see it as the same concept as catcls_class_install(), you need to remove the quotes from the quoted parts like "CTV_CLASS_NAME" in the existing code.
It can be seen as an error in the existing code.